playout/libretime_playout/player/fetch.py

```diff
@@ -138,6 +138,9 @@ def set_bootstrap_variables(self):
                 message_format=preferences.message_format,
                 message_offline=preferences.message_offline,
                 input_fade_transition=preferences.input_fade_transition,
+                default_crossfade_duration=preferences.default_crossfade_duration,
+                default_fade_in=preferences.default_fade_in,
+                default_fade_out=preferences.default_fade_out,
```

Comment:
This is not triggered when we change the values from the general preferences panel in the UI. So it only set during boot time.

Currently the only place where the liquidsoap interactive settings get updated is when the stream settings in the UI is updated.

While we might want to move the fade settings inside the stream settings panel, those values are only used while building the schedule and are never passed directly to liquidsoap in the first place. We might want to wait that the fix really works before moving too much code to handle this particular issue.

Reply:
I've found the trigger for the stream settings. I could add it to the "general settings" too.
Are you planing to move away form the php-framework? If so I could also try move the settings to Django and start the trigger there.

Reply:
Yes, we are moving away from the php framework.

The current code to edit the setting is still in the legacy app, so lets keep it in legacy for now.